tests/test-timber-post.php

```diff
@@ -379,6 +379,47 @@ function testPagedContent(){
 			$this->assertEquals($page2, trim(strip_tags( $post->paged_content() )));
 		}
 
+		function testMetaCustomPreFilterDisable(){
```

Comment:
@jarednova These test show a few errors with 2.x. I’ll make a new pull request to fix them.

Reply:
Blast! I was afraid that might happen